### DISCUSSION TITLE
蓝奏云优享版挂在提示failed get objs: failed to list objs: -1: 超时请求

### DISCUSSION THREAD

**Discussion by im-dashan** [AlistGo/alist#6199] [Invalid question]
蓝奏云优享版挂在提示failed get objs: failed to list objs: -1: 超时请求，怎么解决，有什么办法吗？

**Comment by huolanyun** [comment]
我看更新日志已修复，使用linux版本的，可以参考自编译使用

**Comment by huolanyun** [comment]
[Uploading alist.zip…]()
linux 版本 建议centos 8以上版本使用

**Comment by huolanyun** [comment]
[alist.zip](https://github.com/alist-org/alist/files/14566881/alist.zip)
linux 版本 建议centos 8以上版本使用

**Reply by im-dashan** [reply]
是的，已经不会显示-1超时了，但是文件夹id好像不能生效了
